index.js

```diff
@@ -15,6 +15,16 @@ let reverse = (s) => {
   return o;
 };
 
+exports.createS3UploadParams = (bucketname, data) => {
+  // The synctoken is unix epoch time, using a regex is faster than parsing JSON :)
+  let regexToken = data.match(/syncToken\D+([0-9]+)/)[1];
```

Comment:
Codacy has a fix for the issue: [Unexpected character class '[0-9]'. Use '\d' instead.](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
  let regexToken = data.match(/syncToken\D+(\d+)/)[1];
```